qiita_ware/search.py

```diff
@@ -0,0 +1,101 @@
+from collections import Counter, defaultdict
+
+from future.utils import viewvalues, viewitems
+
+from qiita_db.study import Study
+from qiita_db.data import ProcessedData
+
+
+def count_metadata(results, meta_cols):
+    """Counts the metadata found in a search, and returns these counts
+
+    Parameters
+    ----------
+    results : dict of lists of list
+        results in the format returned by the qiita_db search obj
```

Comment:
this doesn't indicate what expectations there are about the keys and values, just types. if this method should only ever under any circumstance be called from the search object, then it should be a private member method of that object. otherwise, you have to assume that you do not know who the caller is. this is important both for the documentation here, and wide possibility for a bug on line 36